src/pk/ecc/ecc_verify_hash.c

```diff
@@ -116,9 +126,6 @@ int ecc_verify_hash_raw(      void   *r, void   *s,
       if ((err = ltc_mp.ecc_ptmul(u1, mG, mG, m, 0)) != CRYPT_OK)                                       { goto error; }
       if ((err = ltc_mp.ecc_ptmul(u2, mQ, mQ, m, 0)) != CRYPT_OK)                                       { goto error; }
 
-      /* find the montgomery mp */
```

Comment:
It was mistake, I was removing some code related to curves with custom A and removed also this one.